src/aurelia-loader-webpack.ts

```diff
@@ -127,7 +84,6 @@ export class WebpackLoader extends Loader {
       const registry = __webpack_require__.c;
       const cachedModuleIds = Object.getOwnPropertyNames(registry);
       cachedModuleIds
-        .filter(moduleId => typeof moduleId === 'string')
```

Comment:
Why remove this `filter`? We only care about those modules which are strings and don't want to continue executing logic on numbered modules.